python_training/test_me.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from .pycalculator import Calculator
+
+invalid_parameters = [('a', 'b'), (1, 'a'), (1, None), ({'a': 1}, 2)]
```

Comment:
Since this is used only once in your tests, it's better to use `paramterize` for this specific test (and define it there).